Framework/Algorithms/inc/MantidAlgorithms/MultipleScatteringCorrectionDistGraber.h

```diff
@@ -0,0 +1,49 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidAlgorithms/DllConfig.h"
+#include "MantidGeometry/IDTypes.h"
+#include "MantidGeometry/Objects/IObject.h"
+
+namespace Mantid {
+
+namespace Geometry {
+class IDetector;
```

Comment:
Not the case here, but there is a `MantidGeometry/IDetector_fwd.h` which also includes the smart pointers as foward declares.